packages/dds/tree/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -237,11 +236,16 @@ function handleCurrAttach<T>(
     factory: MarkListFactory<T>,
     lineageRequests: LineageRequest<T>[],
     offset: number,
-    reattachRevision: RevisionTag | undefined,
+    baseRevision: RevisionTag | undefined,
 ) {
     const rebasedMark = clone(currMark);
-    if (reattachRevision !== undefined) {
-        tryRemoveLineageEvent(rebasedMark, reattachRevision);
+
+    // If the changeset we are rebasing over has the same revision as an event in rebasedMark's lineage,
+    // we assume that the base changeset is the inverse of the changeset in the lineage, so we remove the lineage event.
+    // TODO: Handle cases where the base changeset is a composition of multiple revisions.
+    // TODO: Don't remove the lineage event in cases where the event isn't actually inverted by the base changeset.
```

Comment:
Can you give an example of such a scenario? Is this about inserts not undoing deletes?